src/mltk/predictor/gam/GA2MLearner.java

```diff
@@ -68,6 +69,9 @@
 
 		@Argument(name = "-l", description = "learning rate (default: 0.01)")
 		double learningRate = 0.01;
+		
+		@Argument(name = "-p", description = "maximum number of pairs")
```

Comment:
FAST only ranks all pairs. The typical usage of this code is run FAST to get the ranking of all pairs, use `head` command to get the top K pairs and save it to a new file, feed the new file to GA2MLearner.